ETWAnalyzer/EventDump/DumpFile.cs

```diff
@@ -396,6 +399,7 @@ private List<MatchData> AggregateByDirectory(List<MatchData> data, int level)
                             fileSetSecurityCount += (long)    (stats.SetSecurity?.Times?.Count).GetValueOrDefault();
                             fileDeleteCount += (long)         (stats.Delete?.Count).GetValueOrDefault();
                             fileRenameCount += (long)         (stats.Rename?.Count).GetValueOrDefault();
+                            fileOpenCloseTimeInus += (long)   (stats.Open?.Durationus).GetValueOrDefault() + (stats.Close?.Durationus).GetValueOrDefault();
```

Comment:
You should not need it since you already have open and close times there

Reply:
resolved